config/locales/hyrax.en.yml

```diff
@@ -727,6 +730,7 @@ en:
         keyword:                   "Keyword"
         lease_expiration_date:     'until'
         related_url:               "Related URL"
+        title:                     "Title"
```

Comment:
That's fine, I can move it if you prefer.  I just thought it would be better to catch all of the untranslated titles at once. And additionally, since title hints were under default, I thought it made sense to put title under default as well for consistency purposes.